qa-tests-backend/src/test/groovy/CSVTest.groovy

```diff
@@ -306,6 +310,7 @@ class CSVTest extends BaseSpecification {
     }
 
     @EqualsAndHashCode(includeFields = true)
+    @ToString(includeFields = true)
```

Comment:
Nit: Would these cause the log to be very noisy if all cves are being logged?

Reply:
That is right, I should only include id and cvss since they are relevant to debugging.